core/store/src/trie/split_state.rs

```diff
@@ -43,7 +26,7 @@ impl ShardTries {
         &self,
         state_roots: &HashMap<ShardUId, StateRoot>,
         changes: StateChangesForSplitStates,
-        account_id_to_shard_id: &dyn Fn(&AccountId) -> ShardUId,
+        account_id_to_shard_uid: &dyn Fn(&AccountId) -> ShardUId,
```

Comment:
totally not for this PR but I am strongly against using lambdas like that
it makes it really hard to read and follow the code execution

Reply:
I'll try a possible cleanup as part of a separate PR.